src/routes/resources/+page.svelte

```diff
@@ -146,23 +148,25 @@
             class="input input-bordered input-md w-[11rem]"
             placeholder="Chapter (e.g. 2, 1-5)"
         />
-        <div class="flex flex-col space-y-2">
+        <div class={shouldShowNotApplicable ? 'flex flex-col space-y-2' : 'flex'}>
```

Comment:
This centers the "Show Only Published" checkbox if the "Show Only N/A" checkbox is not displayed.  If there is a better way to accomplish this please comment here.